kogito-kafka-quickstart-quarkus/README.md

```diff
@@ -16,15 +16,66 @@ This example shows
 	
 <p align="center"><img width=75% height=50% src="docs/images/process.png"></p>
 
+* Diagram Properties
+<p align="center"><img src="docs/images/diagramProperties.png"></p>
+
+* Diagram Properties
+<p align="center"><img src="docs/images/diagramProperties2.png"></p>
+
+* Diagram Properties
+<p align="center"><img src="docs/images/diagramProperties3.png"></p>
+
+* Start Message
+<p align="center"><img src="docs/images/startMessage.png"></p>
+
+* Start Message Assignments
+<p align="center"><img src="docs/images/startMessageAssignments.png"></p>
+
+* Process Traveler Business Rule
+<p align="center"><img src="docs/images/processTravelerBusinessRule.png"></p>
+
+* Process Traveler Business Rule
+<p align="center"><img src="docs/images/processTravelerBusinessRule2.png"></p>
+
+* Process Traveler Gateway
+<p align="center"><img src="docs/images/processedTravelerGateway.png"></p>
+
+* Process Traveler Gateway Yes Connector
+<p align="center"><img src="docs/images/processedTravelerYesConnector.png"></p>
+
+* Process Traveler Gateway No Connector
+<p align="center"><img src="docs/images/processedTravelerNoConnector.png"></p>
+
+* Log Traveler Script Task
+<p align="center"><img src="docs/images/logTravelerScriptTask.png"></p>
+
+* Skip Traveler Script Task
+<p align="center"><img src="docs/images/skipTravelerScriptTask.png"></p>
+
+* Processed Traveler End Message
+<p align="center"><img src="docs/images/processedTravelerEndMessage.png"></p>
```

Comment:
seems like application.properties is set to `processedtravellers`
Please update bpmn on End message with Message set to `processedtravellers` instead of `handletravellers`

Reply:
Fixed @radtriste